addons/website_blog/views/snippets/snippets.xml

```diff
@@ -9,7 +9,7 @@
 
 <template id="snippet_options" inherit_id="website.snippet_options" name="Blog snippet options">
     <xpath expr="." position="inside">
-        <div data-js="BlogPostTagSelection" data-selector=".o_wblog_post_page_cover" data-target="#o_wblog_post_name">
+        <div data-js="BlogPostTagSelection" data-selector=".o_wblog_post_page_cover" data-target="#o_wblog_post_name" data-no-check="true">
```

Comment:
@xO-Tx 

> You can also remove the XML attribute from you diff here since you restored the JS version +1

I said earlier: "In fact it would already have been good to do it in the original PR (adding a new attribute is always fine) but not important." -> The JS is required IMO (fixing existing stable users) but the XML update at the same time is also fine in this case IMO (adding such a new attribute is very low risk) :wink: